test/common/testlib.py

```diff
@@ -42,7 +42,10 @@
 import gzip
 
 import tap
-import testvm
+try:
+    from machine_core import testvm
+except ImportError:
+    import testvm
```

Comment:
For the record: Once we bump the test/ version in external projects, this can be cleaned up.